src/Keyboard/Keyboard.php

```diff
@@ -24,16 +24,13 @@
  * ];
  * </code>
  *
- * @method $this setResizeKeyboard($boolean)     Optional. Requests clients to resize the keyboard vertically for
- *         optimal fit.
- * @method $this setOneTimeKeyboard($boolean)    Optional. Requests clients to hide the keyboard as soon as it's been
- *         used.
- * @method $this setSelective($boolean)          Optional. Use this parameter if you want to show the keyboard to
- *         specific users only.
+ * @method $this setResizeKeyboard($boolean)     Optional. Requests clients to resize the keyboard vertically for optimal fit.
+ * @method $this setOneTimeKeyboard($boolean)    Optional. Requests clients to hide the keyboard as soon as it's been used.
+ * @method $this setSelective($boolean)          Optional. Use this parameter if you want to show the keyboard to specific users only.
  */
 class Keyboard extends Base
 {
-    /** @var bool Make an Inline Keyboard */
+    /** @type bool Make an Inline Keyboard */
```

Comment:
Because my find and replace regex string for the others ones caught it by mistake!!

Reverted